plugins/module_utils/config_utils.py

```diff
@@ -435,8 +435,17 @@ def set(self, value: str, setting: str) -> None:
             _setting: Element = self._config_xml_tree.find(xpath)
 
         # If the element is present we will verify it's .text value
-        elif _setting.text in [None, "", " "]:
-            raise NotImplementedError("Currently only text settings supported")
+        elif _setting.text is None or _setting.text.strip() == "":
+            # check if setting has children
+            if (
+                len(_setting.items()) > 0
+                or str(ElementTree.tostring(_setting)).count(_setting.tag) > 1
+            ):
+                # Theoretical Edge case. If a Node were 'intended' to be a parent node,
+                # but has no children, then it would be written as <parent></parent>.
+                # As opposed to children nodes with None value, which are
+                # written as <child />.
+                raise AttributeError(f"Cannot assign value to parent node {_setting}")
```

Comment:
Just a minor rewording suggestion
```suggestion
                raise AttributeError(f"Cannot assign value to node ({_setting}) with child elements.")
```